securepy/stdio.py

```diff
@@ -113,58 +127,67 @@ def __enter__(self) -> None:
 
     def __exit__(self, exc_type, exc_val, exc_tb) -> None:
         """
-        Restore the normal printing STDOUT/STDERR capabilities.
+        Restore the normal STDOUT/STDERR/STDIN capabilities.
         """
         self.restore_std()
 
-    def __call__(self, func: t.Callable) -> t.Any:
+    def __call__(self, func: t.Callable, stdin: t.Optional[str] = None) -> t.Any:
```

Comment:
This would actually fail, there are 2 options to do this, either with 3 functions, with call only taking in the `stdin` and returning a function that takes in `func` which then returns the `inner` function as defined here, or just remove this and only work with pure decorator without any arguments for `stdin`. 

Personally, I think we don't strictly need this ability for decorator as it isn't in the context manager either.

Reply:
I will fix it, as it feels unnecessary for a decorator, and would be difficult to keep clean.

Reply:
Fixed